.github/workflows/release-python-client.yaml

```
@@ -0,0 +1,55 @@
+name: Publish Python 🐍 distributions 📦 to PyPI and TestPyPI
+
+on: push
+jobs:
+  build-n-publish:
+    name: Build and publish Python 🐍 distributions 📦 to PyPI and TestPyPI
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@master
+      - name: Set up Python 3.7
+        uses: actions/setup-python@v1
+        with:
+          python-version: 3.7
+      - name: set semver
+        run: |
+          CURRENT_SEMVER=$(curl -s -X GET "https://jade.datarepo-dev.broadinstitute.org/configuration" -H "accept: application/json" | jq -r '.semVer|rtrimstr("-SNAPSHOT")')
+          echo "CURRENT_SEMVER=$(echo ${CURRENT_SEMVER})" >> $GITHUB_ENV
+      - name: Generate Python client with openapi-generator
+        # openapi-generator-cli generate -i src/main/resources/api/data-repository-openapi.yaml -g python -o ../data-repo-client --package-name data_repo_client
+        # -c /github/workspace/${{ inputs.config-file }} \
+        run: |
+          docker run -u 1001 --rm --workdir /github/workspace -v "$GITHUB_WORKSPACE":"/github/workspace" \
+          openapitools/openapi-generator-cli:v4.3.1 generate \
+          -i /github/workspace/src/main/resources/api/data-repository-openapi.yaml \
+          -g python \
+          -o /github/workspace/data-repo-client \
+          --additional-properties=projectName=data-repo-client,packageName=data_repo_client,packageVersion=${CURRENT_SEMVER} \
```

Follow on change to the previous comment:
```suggestion
          --additional-properties=projectName=data-repo-client,packageName=data_repo_client,packageVersion=${{ steps.set_semver.outputs.CURRENT_SEMVER }} \
```

---

It might be beneficial to use an action here instead of a Docker container, for example with https://github.com/craicoverflow/openapi-generator-generate-action